backend/clubs/views.py

```diff
@@ -1834,11 +1839,14 @@ def list(self, request, *args, **kwargs):
 
     def perform_destroy(self, instance):
         """
-        Manually delete uploaded asset instances because of PostgreSQL integrity rules.
+        Set archived boolean to be True so that the club appears to have been deleted
         """
-        for asset in instance.asset_set.all():
-            asset.delete()
-        instance.delete()
+
+        club = self.get_object()
```

Comment:
@cphalen Have you tested this code? I have a feeling that `get_object` wouldn't exist/be appropriate and you would want to use `instance` instead.

Reply:
Given the test cases passes I think you'd be fine, but I would use `instance` just in case.

Reply:
Ok awesome good catch — thanks Eric!